src/components/App.css

```diff
@@ -0,0 +1,252 @@
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+nav,
+ul {
+  display: flex;
+  align-items: center;
+}
+
+ul {
+  list-style-type: none;
+  width: 60%;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.panel-bg {
+  display: flex;
+  justify-content: space-around;
```

Comment:
- [ ] `justify-content: space-around;` is used 6 different places. Kindly do the same for this and other duplicated rules.